RHTE-LAB.adoc

```diff
@@ -0,0 +1,844 @@
+Compressing objects: 100% (9/9), done.
+Writing objects: 100% (10/10), 2.10 KiB | 0 bytes/s, done.
+Total 10 (delta 5), reused 1 (delta 0)
+remote: Resolving deltas: 100% (5/5), completed with 5 local objects.
+To https://github.com/__GITHUB_ACCOUNT__/operationalizing-openshift-lab
+   a46e90f..3fa37a4  master -> master
+--------------------------------------------------------------------------------
++
+NOTE: If your GitHub account is configured for two-factor authentication then you will need to configure a
+link:https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line[GitHub personal access token]
+to authenticate for the `git push` command.
+
+Configure Eunomia
+~~~~~~~~~~~~~~~~
+
+Configure Eunomia using `openshift-applier`:
+
+. Install `openshift-applier` into the `galaxy` directory:
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *ansible-galaxy install -r requirements.yml -p galaxy*
+- extracting openshift-applier to /home/lab-user/operationalizing-openshift-lab/galaxy/openshift-applier
+- openshift-applier (master) was installed successfully
+- extracting self to /home/lab-user/operationalizing-openshift-lab/galaxy/self
+- self (master) was installed successfully
+--------------------------------------------------------------------------------
+
+. Set and export the `LDAP_BIND_PASSWORD` environment variable:
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *export LDAP_BIND_PASSWORD=xxxxxx*
+--------------------------------------------------------------------------------
++
+NOTE: The LDAP bind password is included in the authentication unit of the "Red Hat OpenShift Container Platform 4 Configuration" course in the learning management system.
+
+. Configure cluster secrets by running `openshift-applier` with the objects tagged "cluster-secrets":
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml \
+  -e include_tags=cluster-secrets -e exclude_tags=*
+--------------------------------------------------------------------------------
++
+NOTE: We must override `exclude_tags` to set cluster secrets because this variable is set in `.applier/group_vars/seed-hosts/main.yml` to exclude the `cluster-secrets` tag.
+
+. Configure Eunomia:
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *ansible-playbook -i .applier/ galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml \
+  -e include_tags=gitops*
+--------------------------------------------------------------------------------
+
+. Check Eunomia Configuration
++
+[subs=+quotes]
+--------------------------------------------------------------------------------
+$ *oc get gitopsconfig cluster-config -n cluster-config -o yaml*
```

Comment:
ok... I think we still have some cleanup things we are missing. when I get to this point in the lab, my `GitOpsConfig` looks like this:
```
$ oc get gitopsconfig cluster-config -n cluster-config -o yaml
apiVersion: eunomia.kohls.io/v1alpha1
kind: GitOpsConfig
metadata:
  annotations:
    gitopsconfig.eunomia.kohls.io/initialized: "true"
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"eunomia.kohls.io/v1alpha1","kind":"GitOpsConfig","metadata":{"annotations":{},"name":"cluster-config","namespace":"cluster-config"},"spec":{"resourceDeletionMode":"None","resourceHandlingMode":"None","serviceAccountRef":"eunomia-runner","templateProcessorImage":"quay.io/kohlstechnology/eunomia-applier:v0.0.1","templateSource":{"contextDir":"","ref":"master","uri":"https://github.com//operationalizing-openshift-lab"},"triggers":[{"type":"Change"}]}}
  creationTimestamp: "2019-09-13T14:13:28Z"
  finalizers:
  - eunomia-finalizer
  generation: 4
  name: cluster-config
  namespace: cluster-config
  resourceVersion: "1529485"
  selfLink: /apis/eunomia.kohls.io/v1alpha1/namespaces/cluster-config/gitopsconfigs/cluster-config
  uid: a8a0af54-d630-11e9-a440-02ae9dd18d3a
spec:
  parameterSource:
    contextDir: .
    ref: master
    uri: https://github.com/redhat-cop/operationalizing-openshift-lab
  resourceDeletionMode: None
  resourceHandlingMode: None
  serviceAccountRef: eunomia-runner
  templateProcessorImage: quay.io/kohlstechnology/eunomia-applier:v0.0.1
  templateSource:
    contextDir: ""
    ref: master
    uri: https://github.com//operationalizing-openshift-lab
  triggers:
  - type: Change
```